src/components/Sidebar/SidebarContext.spec.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {
+  SidebarContextProvider,
+  SidebarContextConsumer
+} from './SidebarContext';
+
+describe('SidebarContext', () => {
+  describe('styles', () => {
+    it('should render and match the snapshot', () => {
+      const actual = create(
+        <SidebarContextProvider>
+          <SidebarContextConsumer>
+            {({ isSidebarOpen, toggleSidebar }) => (
+              <div
+                isSidebarOpen={isSidebarOpen}
+                toggleSidebar={toggleSidebar}
+              />
+            )}
+          </SidebarContextConsumer>
+        </SidebarContextProvider>
+      );
+      expect(actual).toMatchSnapshot();
+    });
+  });
+
+  it('should change Provider open state when using toggleSidebar', () => {
+    const wrapper = mount(
+      <SidebarContextProvider>
+        <SidebarContextConsumer>
+          {({ toggleSidebar }) => (
+            <button data-selector="child" onClick={toggleSidebar} />
+          )}
+        </SidebarContextConsumer>
+      </SidebarContextProvider>
+    );
+    const actual = wrapper.find('[data-selector="child"]');
+
+    actual.simulate('click');
+    expect(wrapper.instance().state.isSidebarOpen).toEqual(true);
+  });
+
+  describe('accessibility', () => {
```

Comment:
Same goes for the accessibility test. This is a pure logic component that doesn't render any markup which could be checked for its accessibility.